generic_config_updater/patch_sorter.py

```diff
@@ -921,6 +981,8 @@ def validate(self, move, diff):
                 for required_path, required_value in data[path]:
                     current_value = self.identifier.get_value_or_default(current_config, required_path)
                     simulated_value = self.identifier.get_value_or_default(simulated_config, required_path)
+                    if simulated_value == None: # Simulated config does not have this value at all.
```

Comment:
If simulated config does not have the value, it means the move is deleting its parent. in such case skip and assume the mvoe is valid. Let other validators decide.